portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -45,9 +52,25 @@ public DataServlet() {
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     List<Comment> list = new ArrayList<>();
```

Comment:
Define variables close to where they are used.
(https://google.github.io/styleguide/javaguide.html#s4.8.2-variable-declarations)